addons/portal/static/src/chatter/core/portal_chatter.xml

```diff
@@ -4,7 +4,7 @@
 <t t-name="portal.Chatter">
     <div t-if="state.thread" class="o-mail-Chatter w-100 h-100 flex-grow-1 d-flex pt-2" t-att-class="{ 'row':props.twoColumns, 'flex-column':!props.twoColumns }" t-on-scroll="onScrollDebounced" t-ref="root">
         <div t-if="props.composer" class="o-mail-Chatter-top position-sticky top-0" t-att-class="{ 'col-lg-6':props.twoColumns, 'bg-view':env.inPortalChatter }" t-att-style="(!props.twoColumns and env.inPortalChatter) and 'top: -1px !important; margin-top:-15px; padding-top: 20px'" t-ref="top">
-            <Composer composer="state.thread.composer" autofocus="true" mode="'extended'" onPostCallback.bind="onPostCallback" dropzoneRef="rootRef" t-key="props.threadId"/>
+            <Composer composer="state.thread.composer" autofocus="env.inPortalChatter ? false : true" mode="'extended'" onPostCallback.bind="onPostCallback" dropzoneRef="rootRef" t-key="props.threadId"/>
```

Comment:
Could be worth renaming `inPortalChatter` to `inFrontendPortalChatter`, because this is strange to have `portal.Chatter` template with conditionals "inPortalChatter"